src/core/reducers/installations.js

```diff
@@ -20,14 +20,11 @@ export type InstalledAddon = {
   error?: string,
   guid: $PropertyType<AddonType, 'guid'>,
   needsRestart?: boolean,
-  // TODO: merge with core/constants.validInstallStates
```

Comment:
Did you consider doing this? You could use `$ObjMap` but it would require a change to `validInstallStates` in `core/constants`.

Reply:
I double checked that both were in sync, and decided to use `$PropertyType<InstalledAddon, 'status'>`.

Reply:
How do you mean? I still see duplication:

This:

https://github.com/mozilla/addons-frontend/blob/21a551e4e993e8b5b9b012588537cdea58eb138b/src/core/reducers/installations.js#L23-L35

Versus:

https://github.com/mozilla/addons-frontend/blob/21a551e4e993e8b5b9b012588537cdea58eb138b/src/core/constants.js#L16-L30


The comment you removed was about making them share a common type.

Reply:
The constant is still being used in the `InstallSwitch`. I cannot remove the constant entirely yet.

Reply:
Yes!